pymc_extras/statespace/models/utilities.py

```diff
@@ -19,14 +20,23 @@
 
 
 def make_default_coords(ss_mod):
-    coords = {
-        ALL_STATE_DIM: ss_mod.state_names,
-        ALL_STATE_AUX_DIM: ss_mod.state_names,
-        OBS_STATE_DIM: ss_mod.observed_states,
-        OBS_STATE_AUX_DIM: ss_mod.observed_states,
-        SHOCK_DIM: ss_mod.shock_names,
-        SHOCK_AUX_DIM: ss_mod.shock_names,
-    }
+    ALL_STATE_COORD = Coord(dimension=ALL_STATE_DIM, labels=ss_mod.state_names)
```

Comment:
This function should be eliminated in favor of the method `CoordsInfo.defaults_from_model`